xml/vt_installation.xml

```diff
@@ -14,106 +14,160 @@
+    <imageobject role="html">
+     <imagedata fileref="virt-system-roles.png" width="75%"/>
+    </imageobject>
+   </mediaobject>
+  </figure>
+  <para>
+   Here you can select one of <guimenu>KVM Virtualization Host</guimenu> or
+   <guimenu>Xen Virtualization Host</guimenu> roles. The appropriate software
+   selection and setup will be automatically aligned during the host
+   installation.
+  </para>
+  <tip>
+   <para>
+    Both virtualization system roles will create a dedicated
+    <filename>/var/lib/libvirt</filename> partition, and enable the firewall
+    and &kdump; services.
+   </para>
+  </tip>
  </sect1>
- <sect1 xml:id="sec-vt-installation-xen">
-  <title>Installing &xen;</title>
 
+ <sect1 xml:id="vt-installation-yast">
+  <title>Running the <package>yast2-vm</package> module</title>
   <para>
-   To install &xen; and &xen; tools, proceed as follows:
+   Depending on the scope of the installation, none of the virtualization tools
+   may be installed on your system. They will be automatically installed when
+   configuring the hypervisor with the &yast; module <menuchoice>
+   <guimenu>Virtualization</guimenu> <guimenu>Install Hypervisor and
+   Tools</guimenu></menuchoice>. In case this module is not available in &yast;,
+   install the package <package>yast2-vm</package>.
   </para>
 
-  <procedure>
-   <step>
-    <para>
-     Start &yast; and choose <menuchoice>
-     <guimenu>Virtualization</guimenu> <guimenu>Install Hypervisor and
-     Tools</guimenu></menuchoice>.
-    </para>
-   </step>
-   <step>
-    <para>
-     Select <guimenu>&xen; server</guimenu> for a minimal installation of
-     &xen; tools. Select <guimenu>&xen; tools</guimenu> if a
-     &libvirt;-based management stack is also desired. Confirm with
-     <guimenu>Accept</guimenu>.
-    </para>
-   </step>
-   <step>
-    <para>
-     To enable normal networking for the &vmguest;, using a
-     network bridge is recommended. &yast; offers to automatically
-     configure a bridge on the &vmhost;. Agree to do so by choosing
-     <guimenu>Yes</guimenu>, otherwise choose <guimenu>No</guimenu>.
-    </para>
-   </step>
-   <step>
-    <para>
-     After the setup has been finished, you need to reboot the machine with
-     the <emphasis>&xen; kernel</emphasis>.
-    </para>
-    <tip>
-     <title>Default boot kernel</title>
+  <sect2 xml:id="sec-vt-installation-kvm">
+   <title>Installing &kvm;</title>
+
+   <para>
+    To install &kvm; and KVM tools, proceed as follows:
+   </para>
+
+   <procedure>
+    <step>
      <para>
-      If everything works as expected, change the default boot kernel with &yast; and make the &xen;-enabled kernel the default.
-      For more information about changing the default kernel, see <xref linkend="sec-grub2-yast2-config"/>.
+      Verify that the <package>yast2-vm</package> package is installed. This package
+      is &yast;'s configuration tool that simplifies the installation of virtualization
+      hypervisors.
```

Comment:
```suggestion
      is &yast;'s configuration tool that simplifies the installation of hypervisors.
```

Reply:
There's no other kind of hypervisor than a virtualization hypervisor, so I think we can drop the qualifying adjective here.